public/resources/fontello-24c5e6ad/demo.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
```

Comment:
Not sure if all these files are needed or not. I can't imagine we'd want this on prod. /cc @ericawright 

Same applies to the LICENSE and README files above. Not sure if it's _good enough(tm)_ to include the fontello licensing information in the root README.md and not bundle those files in the /public/ dir.